client/client-lib/src/lib.rs

```diff
@@ -450,7 +450,7 @@ impl Client<UserClientConfig> {
                 absolute_timelock as u32,
                 &mut rng,
             )
-            .await?;
+            .unwrap();
```

Comment:
‘?’ Will propagate errors to the caller, while ‘.unwrap()’ will just panic and kill the program. We should keep the ‘?’

Reply:
Fixed by adding ?